blocks/recent-posts/recent-posts.css

```diff
@@ -0,0 +1,98 @@
+.blog main .block.recent-posts[data-block-status="loaded"] {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+}
+
+.blog main .block.recent-posts[data-block-status="loaded"] .blog-card > .card-content > .blog-link h5 {
+    font-family: 'Playfair Display', serif;
+    font-size: 18px;
+    color: var(--text-color);
+  }
+
+  .blog main .block.recent-posts[data-block-status="loaded"] .blog-card > .card-content:hover > .blog-link h5 {
+    color: #705959;
+  }
+
+  .blog main .block.recent-posts[data-block-status="loaded"] .blog-card > .card-content:hover a {
+    text-decoration: none;
+  }
+
+  .blog main .block.recent-posts[data-block-status="loaded"] .blog-card img {
+    object-fit: cover; 
+      height: 100%;
+      width: 100%;
+ }
+
+ .blog main .columns > div {
+    display: flex;
+    flex-direction: column;
+    font-size: 18px;
+    font-family: 'GFS Didot', serif;
+    font-style: italic;
+ }
+
+ .blog main .columns.block > div > div:first-child:hover {
+    color: #705959;
+ }
+
+ .blog main .columns.block > div a.button:any-link, button {
+    font-size: 18px;
+    font-family: 'GFS Didot', serif;
+    font-style: italic;
+    color: var(--text-color);
+    background-color: white;
+    font-weight: 0;
+    border: 0;
+ }
+
+ .blog main .columns.block > div a.button:any-link:hover {
+    color: #705959;
+ }
+
+
+@media (min-width: 600px) {
+    .blog main .section.recent-posts-container {
+        margin-bottom: 40px;
+    }
+
+    .blog main .block.recent-posts[data-block-status="loaded"] {
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        gap: 30px;
+        width: 95%;
+        margin: auto;
+    }
+
+ .blog main .block.recent-posts[data-block-status="loaded"] .blog-card {
+    flex: 1 1 0;
+    width: 100%;
+    height: 25vh;
```

Comment:
Replace this height of 25vh with an aspect-ratio: 16/9 instead.
![image](https://github.com/mkclawson/impala/assets/3883795/caaf69da-15db-4964-8f1f-0fcc6e63b730)

Reply:
fixed